x-pack/legacy/plugins/code/public/components/integrations/code_viewer.tsx

```diff
@@ -0,0 +1,171 @@
+      this.loadFile(content!, lang);
+    } catch (e) {
+      this.loadFile(content!);
+    }
+    if (showBlame) {
+      const blames: GitBlame[] = await requestBlame(repo, revision, file);
+      this.loadBlame(blames);
+    }
+    this.setState({ loading: false });
+  }
+
+  public loadBlame(blames: GitBlame[]) {
+    if (this.blameWidgets) {
+      this.destroyBlameWidgets();
+    }
+    if (!this.lineDecorations) {
+      this.lineDecorations = this.ed!.deltaDecorations(
+        [],
+        [
+          {
+            range: new monaco.Range(1, 1, Infinity, 1),
+            options: { isWholeLine: true, linesDecorationsClassName: 'code-line-decoration' },
+          },
+        ]
+      );
+    }
+    this.blameWidgets = blames.map((b, index) => {
+      return new BlameWidget(b, index === 0, this.ed!);
+    });
+  }
+
+  public destroyBlameWidgets() {
+    if (this.blameWidgets) {
+      this.blameWidgets.forEach((bw: BlameWidget) => bw.destroy());
+    }
+    if (this.lineDecorations) {
+      this.ed!.deltaDecorations(this.lineDecorations!, []);
+      this.lineDecorations = null;
+    }
+    this.blameWidgets = null;
+    if (this.resizeChecker) {
+      this.resizeChecker.destroy();
+    }
+  }
+
+  private loadFile(code: string, language: string = 'text') {
+    const container = document.getElementById('codeViewer') as HTMLElement;
+    this.ed = monaco.editor.create(container, {
+      value: code,
+      language,
+      readOnly: true,
+      minimap: {
+        enabled: false,
+      },
+      hover: {
+        enabled: false,
+      },
+      contextmenu: false,
+      selectOnLineNumbers: false,
+      selectionHighlight: false,
+      renderLineHighlight: 'none',
+      scrollBeyondLastLine: false,
+      renderIndentGuides: false,
+      automaticLayout: false,
+      lineDecorationsWidth: this.props.showBlame ? 316 : 16,
+    });
+    this.resizeChecker = new ResizeChecker(container);
+    this.resizeChecker.on('resize', () => {
+      setTimeout(() => {
+        this.ed!.layout();
+      });
+    });
+  }
+
+  renderFileLoadingIndicator = () => {
+    const fileName = this.props.file;
+    return (
+      <EuiPage restrictWidth>
+        <EuiPageBody>
+          <EuiPageContent verticalPosition="center" horizontalPosition="center">
+            <EuiPageContentHeader>
+              <EuiPageContentHeaderSection>
+                <h2>{fileName} is loading...</h2>
+              </EuiPageContentHeaderSection>
+            </EuiPageContentHeader>
+            <EuiPageContentBody>
+              <EuiProgress size="s" color="primary" />
+            </EuiPageContentBody>
+          </EuiPageContent>
+        </EuiPageBody>
+      </EuiPage>
+    );
+  };
+  render() {
+    return (
+      <EuiFlexGroup direction="row" className="codeContainer__blame" gutterSize="none">
+        {this.state.loading && this.renderFileLoadingIndicator()}
+        <div tabIndex={0} className="codeContainer__root" id="codeViewer" />
```

Comment:
The class that applies monaco overrides has been renamed to `codeContainer__monaco` as of #47636, and it no longer applies layout CSS if you were overriding/depending on that.

If you need to add additional style to this element, I would suggest adding another class like `"codeFlyout__code-block"` instead of an ID.